etc/pixels.properties

```diff
@@ -0,0 +1,287 @@
+###### pixels-daemon settings ######
+# pixels.var.dir is where the lock files are created
+pixels.var.dir=/home/pixels/opt/pixels/var/
+# metadata database connection properties
+metadata.db.driver=com.mysql.cj.jdbc.Driver
```

Comment:
Remove this file.
etc/pixels.properties should exist in the installation directory of pixels (i.e., PIXELS_HOME), not the source directory.